modules/apps/portal-search/portal-search-web/src/main/resources/content/Language.properties

```diff
@@ -161,6 +163,7 @@ set-up-its-destination-to-make-it-visible=Set up its destination to make it visi
 site-parameter-name=Site Parameter Name
 sort-by=Sort by
 specific-facet-configuration=Specific Facet Configuration
+tag-facet-portlet-instance-configuration-name=Tag Facet Configuration
```

Comment:
Shouldn't the language key and the value match?

Reply:
It looks like on portal, this description is more abbreviated? I can update it to match.

![image](https://user-images.githubusercontent.com/14063502/74365950-ea966200-4d83-11ea-812d-62c15a890455.png)